ll-time-input.html

```diff
@@ -84,10 +93,31 @@
      * @private
      */
     _isoToDisplay: function (isoStr) {
-      var m = moment(isoStr, 'HH:mm:ss');
-      if (m.isValid()) {
+      var m = this._getValidIsoMoment(isoStr);
+      if (!!m) {
```

Comment:
When I added the _getValidity method, there were going to be 2 copies of the following strings of code: "var m = moment(str, 'HH:mm:ss');" and "m.isValid()" so I moved them into a new method_getValidIsoMoment, which returns a moment only if it can parse a valid one from a string. This way, _isoToDisplay converts invalid iso time strings to either their 'h:mm A' format, or null (which is displayed as a blank string automatically), and _getValidity returns true if any moment is returned (because only valid moments are returned).